src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -46,13 +61,11 @@ const BioAndChat = (props) => {
       const response = await sendSignedDMOtherProfile(signedMParams);
       const newResponse = {...response, members: response?.message?.members};
       await saveChatFromOtherProfile(newResponse, 'sent', true, SIGNED);
-      setDmChat('');
     } catch (error) {
+      setLoadingSendDM(false);
       if (__DEV__) {
         console.log(error, 'error');
       }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling here could be improved. Currently, if an error occurs, `setLoadingSendDM(false);` is called, but `setDmChat('');` is not. This could leave the chat in an inconsistent state. Also, the error is only logged in development mode. It would be better to handle the error in a way that's useful in both development and production environments. Consider adding a user-friendly error message or some other form of error handling for production.

```diff
  } catch (error) {
+   setDmChat('');
    setLoadingSendDM(false);
    if (__DEV__) {
      console.log(error, 'error');
    }
+   else {
+     // Handle error for production environment
+   }
  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->